src/components/DropDownMenu/DropDownMenu.react.js

```diff
@@ -3,7 +3,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
-import DropDownMenu from 'material-ui/DropDownMenu';
+import { DropDownMenu as MuiDropDownMenu } from 'material-ui/DropDownMenu';
```

Comment:
Does this work? It was a default export before. (`import DropDownMenu from 'material-ui/DropDownMenu';`)

Reply:
It is coded as the default with `exports.default` in material-ui, but it is also exported as `exports.DropDownMenu`. It was the only one to work at first, when I changed everything over. The component passes its tests and runs correctly in `usage.py`.